contents/docs/libraries/dotnet/index.mdx

```diff
@@ -0,0 +1,153 @@
+posthog.Capture(
+    "distinct_id",
+    "event_name",
+    personPropertiesToSet: new() { ["name"] = "Max Hedgehog" },
+    personPropertiesToSetOnce: new() { ["initial_url"] = "/blog" }
+);
+```
+
+For more details on the difference between `$set` and `$set_once`, see our [person properties docs](/docs/data/user-properties#what-is-the-difference-between-set-and-set_once).
+
+To capture [anonymous events](/docs/data/anonymous-vs-identified-events) without person profiles, set the event's `$process_person_profile` property to `false`:
+
+```csharp
+posthog.Capture(
+    "distinct_id",
+    "event_name",
+    properties: new() {
+        ["$process_person_profile"] = false
+    }
+)
+```
+
+## Alias
+
+Sometimes, you want to assign multiple distinct IDs to a single user. This is helpful when your primary distinct ID is inaccessible. For example, if a distinct ID used on the frontend is not available in your backend. 
+
+In this case, you can use `alias` to assign another distinct ID to the same user.
+
+```csharp
+await posthog.AliasAsync('current_distinct_id', 'new_distinct_id');
+```
+
+We strongly recommend reading our docs on [alias](/docs/product-analytics/identify#alias-assigning-multiple-distinct-ids-to-the-same-user) to best understand how to correctly use this method.
+
+## Group analytics
+
+Group analytics allows you to associate an event with a group (e.g. teams, organizations, etc.). Read the [Group Analytics](/docs/user-guides/group-analytics) guide for more information.
+
+> **Note:** This is a paid feature and is not available on the open-source or free cloud plan. Learn more on our [pricing page](/pricing).
+
+-   Capture an event and associate it with a group
+
+```csharp
+postHogClient.Capture(
+    "user_distinct_id", 
+    "some_event", 
+    groups: [new Group("company", "company_id_in_your_db")]);
+```
+
+-   Update properties on a group
+
+```csharp
+await postHogClient.GroupIdentifyAsync(
+    type: "company",
+    key: "company_id_in_your_db",
+    name: "Awesome Inc.",
+    properties: new()
+    {
+        ["employees"] = 11
+    }
+);
+```
+
+The `name` is a special property which is used in the PostHog UI for the name of the group. If you don't specify a `name` property, the group ID will be used instead.
+
+## Feature flags
+
+import FeatureFlagsLibsIntro from "../_snippets/feature-flags-libs-intro.mdx"
+
+<FeatureFlagsLibsIntro />
+
+import DotNetFeatureFlagsCode from '../../integrate/feature-flags-code/_snippets/feature-flags-code-dotnet.mdx'
+
+<DotNetFeatureFlagsCode />
+
+### Local evaluation
+
+import LocalEvaluationIntro from "../../feature-flags/snippets/local-evaluation-intro.mdx"
+
+<LocalEvaluationIntro />
+
+For details on how to implement local evaluation, see our [local evaluation guide](/docs/feature-flags/local-evaluation).
+
+## Experiments (A/B tests)
+
+Since [experiments](/docs/experiments/manual) use feature flags, the code for running an experiment is very similar to the feature flags code:
```

Comment:
```suggestion
Since [experiments](/docs/experiments/installation) use feature flags, the code for running an experiment is very similar to the feature flags code:
```